concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3403,7 +3401,7 @@ jobs:
         - get: smoke-tests-timer
           trigger: true
         - get: deployed-healthcheck
-        - get: cf-smoke-tests
+        - get: cf-smoke-tests-release
         - get: paas-cf
           passed: ['post-deploy']
```

Comment:
I'm wondering whether we should unpin `paas-cf` here. As things stand the continuous smoke tests fail once `self-update-pipelines` has run. They only start passing again once `post-deploy` has completed. The error is:
```
missing inputs: cf-smoke-tests
```

The reason for this is that the task that's used comes from the `paas-cf` repo, and therefore (due to the pinning) doesn't match the inputs defined in the pipeline (the version of `paas-cf` being used is still referencing `cf-smoke-tests` instead of `cf-smoke-tests-release`).

I think we should fix this, otherwise we'll get spurious alerts in Pagerduty.

Reply:
I've added a commit to update the pinning. I'll get someone else to sanity-check it.

Reply:
I think it makes sense that we want to use the current version of paas-cf right after the pipeline update as we use three different tasks there. Otherwise the pipeline version would be different from the tasks and it could fail every time when we change them.